frame/transaction-payment/src/lib.rs

```diff
@@ -576,6 +724,49 @@ mod tests {
 		PostDispatchInfo { actual_weight: None, }
 	}
 
+	#[test]
+	fn multiplier_can_fit_all_weight() {
```

Comment:
this is pretty stupid of me to put here... @thiolliere this should be moved to your integrity checks or similar.

Reply:
Fixed